.env

```diff
@@ -2,6 +2,7 @@ VERSION=0.26.0-alpha
 DATABASE_USER=docker
 DATABASE_PASSWORD=docker
 DATABASE_DB=airbyte
+# If you change the config_root here, please change it in MigrationAcceptanceTest and airbyte-server/Dockerfile
```

Comment:
this seems pretty dangerous. 

`MigrationAcceptanceTest`: can't it just read the environment variable? as for `airbyte-server/Dockerfile` i'm less sure about that

`airbyte-server/Dockerfile`: this actually seems like a bit of a problem. the point of it being an env variable is that it is set at _runtime_. with this change we're setting it at build time. so i think there are 2 problems here. Honestly, I'm not sure on what the right answer for this one is. I guess this is the answer to why we had the seed container. I hope there's a better way to solve it, but this is the only thing I can think of so far... Bring back the seed container _concept_. We would use the server container and use that to populate the seeds if they don't exist. Then I think we can still do the rest of the logic you already have for the migration as it is. That way we can remove `COPY build/resources/main/config data/config` from the `airbyte-server/Dockerfile` and not have the env variable problem. Would that work?

```
  seed:
    image: airbyte/server:${VERSION}
    container_name: airbyte-data-seed
    # Pre-populate the volume if it is empty.
    # See: https://docs.docker.com/storage/volumes/#populate-a-volume-using-a-container
    volumes:
      - data:/latest_seeds
```

Reply:
We still would have this problem with seed. In the docker file of seed [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/init/Dockerfile#L8) 
we do `COPY src/main/resources/config seed/config` so if we change it in the docker-compose file, we would have to change it in the docker file as well or vice versa. 

Secondly if we bring back seed, we would have to maintain the seeds in two places, because we want to copy the seeds into the server container as well,  and server docker image will not have access to resources of seed package